src/NestedProportionalAreaChart/HorizontalLegend.js

```diff
@@ -14,6 +14,7 @@ import {
  *
  */
 function HorizontalLegend({
+  numberLabelFormat,
```

Comment:
`numberLabelFormatter`?

Reply:
I thought that name would do but its a function to return a formatted string of a number. Maybe it should be `formatNumberForLabel` ?

Reply:
I'm down with that @karimkawambwa ... Just wanted a name that shows it's a function so `formatNumberForLabel`, `formatNumber`, etc., seem like better names.